src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/AbstractUnitTestingDocumentDifferenceService.cs

```diff
@@ -0,0 +1,180 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Everything in this folder is hte hard fork, with the only changes being that *ALL* the type names have all had 'UnitTesting' added to them.